dts/bindings/led/onnn,ncp5623.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 NXP Semiconductors
+# Copyright (c) 2023 Congipilot Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NCP5623 Triple Output I2C Controlled RGB Led driver
+
+  To test the led use the LED_SHELL application for testing
+
+  The following example maps a RGB led:
+
+  ncp5623c@39 {
+    compatible = "onnn,ncp5623";
+    reg = <0x39>;
+
+    led_0 {
+      label = "RGB LED";
+      index = <0>;
+      color-mapping =
+        <LED_COLOR_ID_RED>,
+        <LED_COLOR_ID_GREEN>,
+        <LED_COLOR_ID_BLUE>;
+    };
+  };
+
+  You can set the RGB colors using:
+
+  led set_color ncp5623c@39 0 20 0 0
+
+  The following example maps 3 individual leds:
```

Comment:
Suggestion:
`The following example defines three single-channel LEDs in the ncp5623 DT node:`